src/main/flight/filter_pt1.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * filter_pt1.c
+ *
+ *  Created on: 24 jun. 2015
+ *      Author: borisb
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <math.h>
+
+#include "common/axis.h"
+
+#include "flight/pid.h"
+#include "flight/filter_pt1.h"
+
+uint16_t cycleTime;
+
+// PT1 Low Pass filter
+float filter_pt1(float input, pt1_state_t* state, int f_cut) {
```

Comment:
generally we don't use `_` characters in function/method name.  how about `filterApplyPt1` instead?

i think perhaps having one single file, called `flight/filter.c` might be good here instead of `filter_this.c` `filter_that.c` since each filter is quite small.

The filter should have a unit test to go with it.  See the baro_unittest.cc for a simple example.